Expensify/App
# Request money placeholder

### Details
cc @deetergp @shawnborton 

Web/desktop views are confusing because it's not an obvious input to put stuff in. 

### Fixed Issues
No issues related to a slack [convo here](https://expensify.slack.com/archives/C03TQ48KC/p1621621835154700)

### Tests
### QA Steps (Web/Desktop only)
1. Select "Request Money" while in an open chat on web/desktop wide screen view
2. Verify there is a placeholder

### Tested On

- Mobile Web
- iOS
- Android

### Screenshots


#### Web
![2021-05-21_10-28-32](https://user-images.githubusercontent.com/32969087/119194905-7a16d000-ba1f-11eb-8de0-734a7b3cf6d8.png)

#### Desktop
![2021-05-21_10-30-28](https://user-images.githubusercontent.com/32969087/119194967-931f8100-ba1f-11eb-9555-c2331ffcd44d.png)
